src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/native.library.packages.targets

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!-- This file contains a map of native-library -> package so that we can harvest package dependencies from native references -->
+
+    <!-- runtime.native.System -->
+    <NativeLibrary Include="System.Native">
```

Comment:
Any reason not to just fold this in with the base line file as another piece of metadata?